### DISCUSSION TITLE
Yay

### DISCUSSION THREAD

**Discussion by Azure-Helper** [TrianguloY/URLCheck#97] [General]
![Screenshot_20220913-203104_YouTube~2](https://user-images.githubusercontent.com/103818282/189937383-a6a19d07-f1db-433f-9abe-1c464b7b0931.png)

**Comment by Ilithy** [comment]
Really nice as a short intro video, but it ignores a lot of URLCheck's capabilities (like checking via VT, url registry etc...), UrlCheck should be integrated into Android because it is so practical and useful, it's very good that it is better known (and recognized)

Thanks for the discovery
